lightning_page_components/DetectAndLaunch/force-app/main/default/aura/detectAndLaunch/detectAndLaunchHelper.js

```diff
@@ -1,8 +1,28 @@
 ({
```

Comment:
is it intentional to have this Detect and Launch file in this pull request with the radio button stuff?

Reply:
Created new PR #1348